pkg/integrations/windows_exporter/config_windows_test.go

```diff
@@ -0,0 +1,61 @@
+//go:build windows
+
+package windows_exporter
+
+import (
+	"github.com/alecthomas/kingpin/v2"
+	"github.com/prometheus-community/windows_exporter/collector"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+	"testing"
+)
+
+func TestConfig(t *testing.T) {
+	built, total := testConfig(t, "")
+	// Default which is windows_exporter defaults minus textfile
+	require.Len(t, built, 7)
+	// Total should be 50
+	require.Len(t, total, 50)
+}
+
+func TestMultipleConfig(t *testing.T) {
```

Comment:
I'm missing a test that `mssql.enabled_classes` is omit here, that the defaults are applied from upstream exporter. Something what the original bug report covered.

Reply:
Good call, adding that test now.